score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -69,18 +71,27 @@ public Long trigger(CompilationArtifact compilationArtifact, Map<String, ? exten
      * This method will trigger the flow in a synchronize matter, meaning only one flow can run at a time.
      * @param compilationArtifact the artifact to trigger
      * @param inputs : flow inputs
+     * @param isQuiet: omits to print task names if set to true
      * @return executionId
      */
+
     @Override
-	public Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties){
+    public Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties, boolean isQuiet){
         //add start event
         Set<String> handlerTypes = new HashSet<>();
-        handlerTypes.add(EventConstants.SCORE_FINISHED_EVENT);
-        handlerTypes.add(EventConstants.SCORE_ERROR_EVENT);
-        handlerTypes.add(EventConstants.SCORE_FAILURE_EVENT);
-        handlerTypes.add(SLANG_EXECUTION_EXCEPTION);
-        handlerTypes.add(EVENT_EXECUTION_FINISHED);
-        handlerTypes.add(EVENT_INPUT_END);
+        if(isQuiet){
+            handlerTypes.add(EVENT_OUTPUT_END);
```

Comment:
This just specifies what events are tracked in both modes. Quiet mode only tracks outputs and the finish event while normal mode also tracks all others. The printOutputs method checks so that only the flow outputs get printed. So step outputs are never printed. Should I also add the exception/error events to be tracked for quiet mode? ATM no errors are printed to CLI in quiet mode.